backend/test/gpml/db/landing_test.clj

```diff
@@ -0,0 +1,59 @@
+(ns gpml.db.landing-test
+  (:require [clojure.test :refer [deftest testing is use-fixtures]]
+            [clojure.java.jdbc :as jdbc]
+            [gpml.db.country :as db.country]
+            [gpml.db.country-group :as db.country-group]
+            [gpml.db.landing :as db.landing]
+            [gpml.db.resource :as db.resource]
+            [gpml.fixtures :as fixtures]
+            [integrant.core :as ig]))
+
+(use-fixtures :each fixtures/with-test-system)
+
+(defn make-resource [title]
+  {:title title
+   :type "Financial Resource"
+   :publish_year 2010
+   :summary "Summary"
+   :value 1000000
+   :image nil
+   :valid_from nil
+   :valid_to nil
+   :geo_coverage_type nil
+   :attachments nil
+   :remarks "Remarks"})
+
+(defn add-resource-data [conn]
+  (db.country/new-country conn {:name "Spain" :iso_code "ESP"})
+  (db.country/new-country conn {:name "India" :iso_code "IND"})
+  (db.country/new-country conn {:name "Indonesia" :iso_code "IDN"})
+  (db.country-group/new-country-group conn {:name "Asia" :type "region"})
+  (db.country-group/new-country-group conn {:name "Europe" :type "region"})
+  (jdbc/insert-multi! conn :country_group_countries
+                      [{:country_group 1 :country 2}
+                       {:country_group 1 :country 3}
+                       {:country_group 2 :country 1}])
+  (db.resource/new-resource conn (make-resource "Resource 1"))
+  (db.resource/new-resource conn (make-resource "Resource 2"))
+  (db.resource/new-resource conn (make-resource "Resource 3"))
+  (jdbc/insert-multi! conn :resource_geo_coverage
+                      [ ;; Resource 1
+                       {:resource 1 :country 1}
+                       {:resource 1 :country 2}
+                       {:resource 1 :country 3}
+                       ;; Resource 2
+                       {:resource 2 :country 3}
+                       ;; Resource 3
+                       {:resource 2 :country_group 1}
+                       {:resource 2 :country_group 2}]))
+
+(deftest test-map-counts
+  (testing "Test map counts for landing page"
+    (let [db-key :duct.database.sql/hikaricp
+          system (ig/init fixtures/*system* [db-key])
+          conn (-> system db-key :spec)
+          _ (add-resource-data conn)
+          landing (db.landing/map-counts conn)]
+      (is (= (:resource (first (filter (fn [x] (= "ESP" (:iso_code x))) landing))) 2))
```

Comment:
`(fn [x] (= "ESP" (:iso_code x)))` can be changed to a use [anonymous reader dispatch](https://clojure.org/reference/reader#_dispatch) so, is equivalent to `#(= "ESP" (:iso_code %))` ... you'll see more often this type of expressions in other code bases

Reply:
Sure, I had something like that in the beginning, when writing out this expression. But, ended up changing it to this form eventually. I'll change it back.